Algorithms.Test/RunningSumArrayTest.cs

```diff
@@ -40,6 +40,21 @@ public void RunningSum_ArrayWithOneElement_ReturnSameArray()
         {
             int[] numsArray = new int[] { 3 };
 
+            var result = _runningSumArray.RunningSum(numsArray);
+            Assert.Equal(numsArray, result);
+        }
+        [Fact]
+        public void RunningSum_ArrayWithNegativeNumber_ReturnSameArray() {
+            int[] numsArray = new int[] { -1000001, 2, 1 };
```

Comment:
nice catch! 
But my question remains... what would happen if I send { -1, -2, -3, 5 }
(I know the code, so I know it would work, but there're missing tests about those cases)